app/mailers/mailshots_mailer.rb

```diff
@@ -9,4 +9,18 @@ def community_launch
     subject = "Forum, Swag, Automated Feedback, and a new Dig Deeper section"
     mail_to_user(@user, subject)
   end
+
+  def company_support_donor
```

Comment:
I've put the two emails here in this class (mailshots), as they're one-off mails. We have a donations mailer too though. Let me know if this is the right place.

Reply:
This is the right place.